x-pack/plugins/uptime/server/graphql/unsigned_int_scalar/resolvers.ts

```diff
@@ -16,10 +16,7 @@ const unsignedIntegerScalar = new GraphQLScalarType({
   },
   parseValue(value) {
     const parsed = parseInt(value, 10);
-    if (parsed < 0) {
-      return null;
-    }
-    return parseInt(value, 10);
+    return parsed < 0 ? null : parsed;
```

Comment:
Hm. I could have just reverted this file. Should I?

Reply:
I kind of like the ternary rather than the nesting.